code/controllers/mc/subsystem.dm

```diff
@@ -13,17 +13,18 @@
 	var/can_fire = TRUE
 
 	// Bookkeeping variables; probably shouldn't mess with these.
-	var/last_fire = 0     // Last world.time we called fire()
-	var/next_fire = 0     // Scheduled world.time for next fire()
-	var/cost = 0          // Average time to execute
-	var/tick_usage = 0    // Average tick usage
-	var/state = SS_IDLE   // Tracks the current state of the ss. Running, paused, etc.
-	var/paused_ticks = 0  // Ticks this ss is taking to run right now.
-	var/paused_tick_usage // Total tick_usage of all of our runs while pausing this run
-	var/ticks = 1         // How many ticks does this ss take to run on avg.
-	var/times_fired = 0   // Number of times we have called fire()
-	var/queued_time = 0   // Time we entered the queue, (for timing and priority reasons)
-	var/queued_priority   // We keep a running total to make the math easier, if it changes priority mid-fire that would break our running total, so we store it here
+	var/initialized = FALSE // Tracks if Initialize() has been called yet.
```

Comment:
All the changed lines below this simply had spaces added before the comments to make them line up with this.